packages/upstream-protobuf/bin/upstream-files.mjs

```diff
@@ -43,7 +43,6 @@ async function main(args) {
             exitUsage();
     }
     stdout.write(protoInclude.files.join(" "));
-    exit(0);
```

Comment:
Unrelated. It's possible that Node.js exits before it has flushed what we wrote to stdout in the line above. Removing this line let's Node.js exit on its own.